extra/benchmark/resnetlike_testset.py

```diff
@@ -20,7 +20,7 @@
 counter = 0
 
 # waffle
-model = nn.Module('mnist_resnetlike', './extra/models/resnetlike/resnetlike_mnist.onnx')
+model = nn.Module('mnist_resnetlike', './extra/models/resnetlike/resnetlike_mnist.onnx', mode ='graph') #graph optipon
```

Comment:
Let's not change the test file. Change this file to how it was again.
Instead let's change the default onnx traverse mode in "waffle/nn/__init__.py/line#12" to graph for this branch!